src/containers/Portal.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import styled from 'styled-components';
+import { ListOfClubs } from "../components/ListOfClubs/ListOfClubs";
+
+const PageWrapper = styled.div`
+    margin: 0 5%;
+    font-family: 'Manrope', sans-serif;
+`;
+
+const HeadingDiv = styled.div`
+    display: flex;
+    justify-content: space-between;
+    flex-direction:row;
+`;
+
+const PendingSection = styled.div`
+    margin: 3% 0;
+`;
+const ApprovedSection = styled.div`
+    margin: 3% 0;
+    text-decoration: underline;
+    cursor: pointer;
+`;
+const TrashSection = styled.div`
+    margin: 3% 0;
+    text-decoration: underline;
+    cursor: pointer;
+`;
+
+const columnTitles = [
+    "Club name:", "Email:", "Date applied:"
+];
+const clubs = [
+    {
+        "clubName": "ADI",
+        "email": "adi@columbia.edu",
+        "dateApplied": "11/20/20",
+        "actions":["Accept","Deny"],
+        "moreInfo": {
+                "description": "Founded in 1972, Multiracial Students Alliance (MSA) provides a space for students who identify as multiracial to gather, socialize, and discuss current personal and political issues that pertain to people with multiple racial and/or ethnic backgrounnds.",
+                "highlights": "1. alsdkfjasdlfkj 2. asdl;fkasdflkjsfd 3. alkdasdlfkjasdflkj"
+            }
+        
+    },
+    {
+        "clubName": "Multiracial Students Alliance",
+        "email": "msa@columbia.edu",
+        "dateApplied": "11/20/20",
+        "actions":["Accept","Deny"],
+        "moreInfo": {
+            "description": "Founded in 1972, Multiracial Students Alliance (MSA) provides a space for students who identify as multiracial to gather, socialize, and discuss current personal and political issues that pertain to people with multiple racial and/or ethnic backgrounnds.",
+            "highlights": "1. alsdkfjasdlfkj 2. asdl;fkasdflkjsfd 3. alkdasdlfkjasdflkj"
+        }
+    }, 
+    {
+        "clubName": "Multiracial Students Alliance",
+        "email": "msa@columbia.edu",
+        "dateApplied": "11/20/20",
+        "actions":["Accept","Deny"],
+        "moreInfo": {
+            "description": "Founded in 1972, Multiracial Students Alliance (MSA) provides a space for students who identify as multiracial to gather, socialize, and discuss current personal and political issues that pertain to people with multiple racial and/or ethnic backgrounnds.",
+            "highlights": "1. alsdkfjasdlfkj 2. asdl;fkasdflkjsfd 3. alkdasdlfkjasdflkj"
+        }
+    }   
+];
+
+
+
+export const Portal = () => {
+    return(
+        <PageWrapper>
+            <HeadingDiv>
+                <h1>Clubs@CU Admin Portal</h1>
+                <p>Log out</p>
+            </HeadingDiv>
+            <PendingSection>
+                <h3>Pending Requests</h3>
+                <ListOfClubs columnTitles={columnTitles} clubs={clubs}></ListOfClubs>
```

Comment:
- As a general rule of thumb, variables such as `clubs` and `columnTitles` in this case should not be hardcoded and passed directly to `<ListOfClubs />`.  

- Instead, it should be passed in as props to `<Portal/>` and then to `<ListOfClubs />`. 
- The reason for this is that during the lifecycle of the Portal component, the clubs in the Pending section might change, such as being moved to the Approved Section or Trash Section. If it's passed directly to ListOfClubs, bypassing Portal, we won't be able to change it later based on user actions.